workflow/tests/base_results/results.csv

```diff
@@ -261,7 +265,7 @@ base-hvac-furnace-gas-room-ac.xml,56.008,56.008,35.247,35.247,20.761,0.0,0.0,0.0
 base-hvac-furnace-wood-only.xml,51.621,51.621,30.966,30.966,0.0,0.0,0.0,20.655,0.0,0.0,0.0,0.0,0.537,0.0,0.0,9.153,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,20.655,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.538,0.0,9.234,0.603,0.0,0.0,0.0,0.0,2105.2,0.0,23.266,0.0,0.0,3.55,3.651,0.514,7.129,0.633,-2.02,0.0,0.0,6.588,-0.116,4.809,0.0,0.729,0.0,5.12,-11.393,0.0,-0.197,-0.714,-0.088,2.068,-0.091,9.061,0.0,0.0,-6.808,-0.055,-1.334,-11.037,-0.219,0.0,-0.0,9.883,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
-base-hvac-furnace-x3-dse.xml,55.205,55.205,35.658,35.658,19.547,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.35,4.16,0.707,9.166,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.547,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.653,8.145,9.234,0.616,0.0,0.0,0.0,0.0,2102.2,2554.9,15.871,10.986,0.0,3.744,3.646,0.513,7.128,0.632,-2.014,0.0,0.0,6.647,-0.118,4.807,0.0,0.729,0.0,0.0,-11.389,0.0,0.075,-0.422,-0.046,2.936,-0.016,9.98,0.0,0.0,-5.631,-0.056,-1.121,-7.045,-0.158,0.0,0.0,9.901,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-hvac-furnace-x3-dse.xml,55.203,55.203,35.656,35.656,19.547,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.347,4.16,0.707,9.166,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,19.547,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.653,8.145,9.234,0.616,0.0,0.0,0.0,0.0,2102.2,2554.9,15.871,10.986,0.0,3.744,3.646,0.513,7.128,0.632,-2.014,0.0,0.0,6.647,-0.118,4.807,0.0,0.729,0.0,0.0,-11.389,0.0,0.075,-0.422,-0.046,2.936,-0.016,9.98,0.0,0.0,-5.631,-0.056,-1.121,-7.045,-0.158,0.0,0.0,9.901,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
@yzhou601 Why is this changing?

Reply:
This is changed by passing wrong arguments to fan disaggregation method when it's a furnace, it is now fixed.